pkg/release/archive.go

```diff
@@ -0,0 +1,33 @@
+package release
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/giantswarm/microerror"
+)
+
+func ArchiveRelease(name, releases, provider string) error {
```

Comment:
This is an extra command I added to move an old release into `archived/` and deregister it.